avm/res/operations-management/solution/main.bicep

```diff
@@ -74,3 +74,25 @@ output resourceGroupName string = resourceGroup().name
 
 @description('The location the resource was deployed into.')
 output location string = solution.location
+
+// =============== //
+//   Definitions   //
+// =============== //
+
+@export()
+type gallerySolutionType = {
```

Comment:
Should be used for the `plan` parameter 😉

Reply:
@AlexanderSehr updated as discussed